### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by neokore**
Related to https://app.clubhouse.io/cartoteam/story/120902/document-widgets-api
Depends on https://github.com/CartoDB/carto-react-lib/pull/10

### CODE REVIEW

File: template-sample-app/template/src/components/views/stores/StoresList.js [hunk 1]
```diff
@@ -57,9 +57,9 @@ export default function StoresList() {
         id='storesByRevenue'
         title='Stores by revenue'
         dataSource={SOURCE_ID}
-        formatter={numberFormatter}
+        yAxisFormatter={numberFormatter}
```
Comment (alasarr) [inline template-sample-app/template/src/components/views/stores/StoresList.js:60]: Not happy with this change. It's not the yAxis, it's the value of the tooltip

**Review by alasarr — CHANGES_REQUESTED** [review]

File: template-sample-app/template/src/components/views/stores/StoresList.js [hunk 2]
```diff
@@ -57,9 +57,9 @@ export default function StoresList() {
...
         xAxisFormatter={currencyFormatter}
-        operation={AggregationTypes.COUNT}
+        operation={AggregationTypes.SUM}
```
Comment (alasarr) [inline template-sample-app/template/src/components/views/stores/StoresList.js:62]: It's not corrent. We don't want to SUM, we want to count the number of stores in the range of the histogram

**Review by alasarr — CHANGES_REQUESTED** [review]

File: template-sample-app/template/src/components/views/stores/StoresList.js (hunk 1 above)
Comment (neokore) [reply template-sample-app/template/src/components/views/stores/StoresList.js:60]: The tooltip is showing the Y axis values...

File: template-sample-app/template/src/components/views/stores/StoresList.js (hunk 2 above)
Comment (neokore) [reply template-sample-app/template/src/components/views/stores/StoresList.js:62]: Sorry, that was a test and I forgot to remove it

Comment (neokore) [reply template-sample-app/template/src/components/views/stores/StoresList.js]: Done

File: template-sample-app/template/src/components/views/stores/StoresList.js (hunk 1 above)
Comment (Josmorsot) [reply template-sample-app/template/src/components/views/stores/StoresList.js:60]: yes @alasarr, this formatter is used in both, tooltip and yAxis

Comment (VictorVelarde) [reply template-sample-app/template/src/components/views/stores/StoresList.js:60]: tooltipFormatter ?

Comment (VictorVelarde) [reply template-sample-app/template/src/components/views/stores/StoresList.js:60]: Agree with Iván. Let's keep it (and adjust the lib accordingly)

**Review by alasarr — APPROVED** [review]

**Pull request merged**